src/js/game/hud/parts/waypoints.js

```diff
@@ -98,6 +100,7 @@ export class HUDWaypoints extends BaseHUDPart {
                 label: null,
                 center: { x: 0, y: 0 },
                 zoomLevel: 3,
+                layer: layers[0],

```

Comment:
You can hardcode this to "regular" or you use `gBuildingRegistry.findByClass(MetaHubBuilding).getLayer()`

Reply:
I did hardcode it to "regular" but vscode and gulp where screaming at me... that its not the Layer type